### DISCUSSION TITLE
[Mobile] Keyboard disappeared after inserting image on iOS

### DISCUSSION THREAD

**Discussion by vunguyen7797** [singerdmx/flutter-quill#453] [Mobile]
My issue is about [Mobile]

I have tried running `example` directory successfully before creating an issue here. 

I encountered a problem that if I inserted an image on iOS device, the keyboard will disappear and never come back. This only happened on iOS and for the image insert only, video insert works well.

To reproduce:
* Select Image insert button on Quill toolbar.
* Select a photo
* After image is inserted into editor, see the error. Trying to tap on editor, the keyboard will not show back up.
* Select Image insert button again and dismiss the pop-up dialog, keyboard should show back up normally again.

https://user-images.githubusercontent.com/17338758/131554740-fbc938d3-742e-45d1-bee3-022e97bc51c2.mov

I tried to investigate but couldn't really figure out why this happened because it didn't happen on Android, thus I created this issue with hope to get some help.

**Comment by renntbenrennt** [comment]
This issue has been discussed and it's caused by image_picker...

**Comment by vunguyen7797** [comment]
> This issue has been discussed and it's caused by image_picker...

Do we have any solutions for this so far?

**Comment by kevinDespoulains** [comment]
I had the same problem and fixed it temporarily by defocusing the focusNode of QuillEditor when picking an image.

For example, you can call `FocusScope.of(context).unfocus();` in the onFilePickImpl callback of the QuillToolbar.

**Comment by YYangyuan** [comment]
I add the code in image_video_utils.dart, and the cursor can appear after the inserted image.

if (imageUrl != null) {
controller.replaceText(index, length, '\n', null);
controller.replaceText(index, length, BlockEmbed.image(imageUrl), null);
}
controller.moveCursorToPosition(index+2);
